ring_benchmark/ring.erl

```diff
@@ -0,0 +1,43 @@
+-module(ring).
+-export([main/1]).
+
+node_loop() ->
+  receive
+    {msg, []} -> ok;
+    {msg, [FirstNode | OtherNodes]} ->
+      FirstNode ! {msg, OtherNodes},
+      node_loop()
+  end.
+    
+
+% N processes, M messages
+main([NArg,MArg]) ->
+  N = arg_to_number(NArg),
+  M = arg_to_number(MArg),
+  [FirstNode | Nodes] = create_processes(N, M),
+  BeforeFistMessage = os:timestamp(),
+  FirstNode ! {msg, Nodes},
+  receive
+    {msg, _} -> ok
```

Comment:
It would be better to receive a more explicit message like `done`, instead of this shape which is intended for the other processes